packages/rancher-logging/generated-changes/overlay/templates/loggings/aks/logging.yaml

```diff
@@ -15,6 +15,9 @@ spec:
     inputTail:
       Tag: "aks"
       Path: "/var/log/azure/kubelet-status.log"
+    security:
+      podSecurityPolicyCreate: true
+      roleBasedAccessControlCreate: true
```

Comment:
Did we decide if we are going to wrap these with `if psp.enabled` conditionals, for users who don't want them. I would say yes because its little work on our side for a failsafe, even if its unlikely to be used.

Reply:
+1, we should do that. Probably would look like...
```yaml
{{- if values.psp.enabled }}
# insert security...
{{- end }}
```

Reply:
@nickgerace I added the ifs